src/templates/app/cashier/payment_methods.jsx

```diff
@@ -471,6 +471,25 @@ const PaymentMethods = () => {
                                     },
                                 ],
                             },
+                            {
+                                id : 'airtm',
+                                row: [
+                                    { text: <PaymentLogo logo='airtm' /> },
+                                    { attributes: { colSpan: 5, className: 'toggler' }, custom_td : <CustomTableData data={[
+                                        { td: it.L('Airtm is a global e-wallet service for money transfers and online payments. For more information, please visit [_1].', '<a href="https://www.airtm.io/#/" target="_blank">https://www.airtm.io/#/</a>') },
```

Comment:
let's remove the `/#` from the display text

Reply:
Sure